igcviewer.js

```diff
@@ -59,12 +59,34 @@
         }
 
         // Show the task declaration if it is present.
-        if (igcFile.task.coordinates.length > 0) {
+        //First two coordinates are always takeoff and start. Last two always finish and landing
+        //So there must be at least 5 for a vailid declaration
+        if (igcFile.task.coordinates.length > 4) {
```

Comment:
Would it be valid to have a declaration with a start and a finish but no turn points (e.g. for a Silver Distance)? If so then the minimum number of points is only 4, not 5.

Also, 'valid' is spelled incorrectly in the comment.